apps/github-actions-cache/src/app/utils.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from 'express'
+import { S3 } from 'aws-sdk'
+import { logger } from '@island.is/logging'
+
+import stream from 'stream'
+
+export const error = (res: express.Response, error: string, status = 400) => {
+  logger.warn(`Sending error "${error}" with status ${status}`)
+  return res.status(status).send({ error })
+}
+
+// In case we want to invalidate all caches in code
+const salt = '3'
+export const getCacheId = (key: string, version: string) =>
+  `${key}:${version}:${salt}`
+
+const uploadChunkSize = 32 * 1024 * 1024
```

Comment:
Is this 32MB? Is this some recommended chunksize you came across?

Reply:
This is the default chunk size in the actions/cache package: https://github.com/actions/toolkit/blob/e2eeb0a784f4067a75f0c6cd2cc9703f3cbc7744/packages/cache/src/options.ts#L59

Reply:
We `could` try to figure it out ourselves, but I'm afraid it could get messy...